ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -148,13 +148,18 @@ def _router_bgp_vrfs(self) -> dict | None:
                     vrfs[vrf_name] = bgp_vrf
                     continue
 
+                vrf_evpn_l3_multicast_enabled = (
+                    default(get(vrf, "l3_multicast.enabled"), get(tenant, "l3_multicast.enabled")) and self._overlay_vtep and self._overlay_evpn
```

Comment:
That seems reasonable.  I had a thought that this would be useful to do but wasn't sure where.  I'll take a look and see if I can fix that as yes it has been used a few times.

Reply:
Fixed in next commit